lib/components/sidebar/card.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Card as AntCard } from 'antd';
+import PropTypes from 'prop-types';
+import styles from './card.css';
+
+const Card = ({ title, extra, cover, bodyStyle }) => (
+  <AntCard
+    {...{
+      className: styles.wrap,
+      bordered: false,
+      title,
+      extra,
+      cover,
+      bodyStyle,
```

Comment:
Do we need `bodyStyle`? Shouldn't the content style itself?

Reply:
I think the bodyStyle is needed, after all we are using the antd component, which contains styles to justify the content, as well as paddings and margins, which may not be suitable for us.

Reply:
The goal is not to make sure that all AntD functions work. Ant Design is only a base for us to build from if we need to. So if there is padding or margins that we do not want, we need to remove them here so our applications do not need to.

We need to make sure that any component that uses any Virtuoso Design System component will always match the style of our design, so we cannot allow inline styles. If there are specific styles that design requires, then we will support those at that time.

Reply:
Ok. Fixed.